mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -249,6 +249,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT to start swimming to avoid suffocation.
 	METHOD method_29495 refreshPositionAfterTeleport (Lnet/minecraft/class_243;)V
 		ARG 1 pos
+	METHOD method_29919 getLeadOffset ()Lnet/minecraft/class_243;
```

Comment:
```suggestion
	METHOD method_29919 getLeashOffset ()Lnet/minecraft/class_243;
```

Reply:
I agree with @YanisBft; only use 'lead' when referring to the item. All other times, even in identifiers, it is named 'leash'.